cg/meta/demultiplex/status_db_storage_functions.py

```diff
@@ -44,7 +44,7 @@ def store_flow_cell_data_in_status_db(
 
 
 def store_sequencing_metrics_in_status_db(flow_cell: FlowCellDirectoryData, store: Store) -> None:
```

Comment:
Is it planned to change the name of this to `store_illumina_sequencing_metrics_in_status_db` at some point?

Reply:
it is planned to remove cg/meta/demultiplex completely in the future and use the illuminapostprocessing service in its place.